### ISSUE TITLE
[Feature request] Drag and drop imporvements

### ISSUE THREAD

**Issue by TMantas** [halower/vue-tree#63]
On drag show position where the draggable node will drop. 
Also I didn't understand how reordering works. I tried to switch nodes (on the same level) but either I moved node as a child or nothing happened.

**Comment by halower** [comment]
yes, the current version does not support exchange of peer locations at the same level, there is not much spare time at the moment, and there are plans behind this feature later
